src/app.js

```diff
@@ -1,21 +1,26 @@
 'use strict';
+/* eslint-disable no-console */
+/* eslint-disable max-len */
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const fs = require('fs');
+const path = require('path');
 
-module.exports = sum;
+const sourcePath = process.argv[2];
+const destinationFolder = process.argv[3];
+const sourceFileName = path.basename(sourcePath);
+const destinationFolderName = path.basename(destinationFolder);
+const destinationPath = path.join(destinationFolder, sourceFileName);
+
+fs.existsSync(destinationFolder, (err) => {
```

Comment:
fs.existsSync does not have a callback function. It should be used in a synchronous manner. If you want to use a callback, consider using fs.access instead.